.pre-commit-hooks.yaml

```diff
@@ -2,7 +2,15 @@
 
 - id: terraform-fmt
   name: Terraform fmt
-  description: Rewrites all Terraform configuration files to a canonical format.
+  description: Rewrites all Terraform configuration files to a canonical format
+  entry: hooks/terraform-fmt.sh
+  language: script
+  files: \.tf$
+  exclude: \.+.terraform\/.*$
+
+- id: terraform-validate
+  name: Terraform validate
+  description: Validates all Terraform configuration files
   entry: hooks/terraform-fmt.sh
```

Comment:
Looks like this is calling the wrong script

Reply:
Oops, my bad, thanks 👍